app/views/components/_heading.html.erb

```diff
@@ -0,0 +1,11 @@
+<%
```

Comment:
Is this heading component identical to https://government-frontend.herokuapp.com/component-guide/heading?
It looks very similar. If it is, is there a plan to move the component into the publishing components so the code is shared?

Reply:
It is indeed the same thing that I copied over. I was going to move it into the publishing components but then I also wondered whether all this was required for a heading tag!